codelab/01_kpt-validate/README.md

```diff
@@ -0,0 +1,32 @@
+# Validate Your Configurations in skaffold Workflow
+
+This codelab introduces how to validate your app configurations before deploying them to the cluster. You are **not** required to know `kpt` or `skaffold` before starting the codelab. 
+
+After this session, you will have obtained more powerful tools to validate and manage your configurations.
+
+## What is kpt?
+
+Kpt is [an OSS tool](https://github.com/GoogleContainerTools/kpt) for Kubernetes packaging, which uses a standard format to bundle, publish, customize, update, and apply configuration manifests.
+
+## What kpt can help you
```

Comment:
I think this should be "What kpt can help you with" or "How kpt can help you"

Reply:
done.